src/app.js

```diff
@@ -4,11 +4,16 @@ import { Provider } from 'react-redux';
 import { configureStore } from './store';
 import { Counter } from './features';
 
+import logo from './svg/cat.svg';
+
 const store = configureStore();
 
 export const App = () => (
-  <Provider store={store}>
-    <h1>Hello, I'm App Component!</h1>
-    <Counter />
-  </Provider>
+  <>
+    <Provider store={store}>
+      <h1>Hello, I'm App Component!</h1>
+      <Counter />
+    </Provider>
+    <img src={logo} alt="" />
+  </> 
 );
```

Comment:
Better place image inside `Provider` and remove fragment.
```
export const App = () => (
 <Provider store={store}>
    <h1>Hello, I'm App Component!</h1>
    <Counter />
    <img src={logo} alt="" />	     
  </Provider>
);
```

Reply:
I recommend creating an Icon component as an example. However, the `img` tag is bad practice. ref: https://medium.com/@gineetmehta/how-to-use-svg-sprites-regular-html-vs-react-9f98948c03f0

Also, I would advise you to investigate this topic. Some articles you can find below 
https://habr.com/ru/post/327700/
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use#Browser_compatibility
https://github.com/JetBrains/svg-sprite-loader/blob/master/2.0.md

The example in the Icon component you can find below

```JSX
import React, { PureComponent } from 'react';
import { string, bool, oneOf, func } from 'prop-types';
import classNames from 'classnames';
import { values } from 'lodash';

import { withRender } from '../shared';
import { SIZES } from '../constants';
import styles from './component.scss';

export class Icon extends PureComponent {
    render() {
        const iconProps = {
            className: classNames(styles.icon, this.props.className, {
                [styles[this.props.size]]: this.props.size
            }),
            onClick: this.props.disabled ? undefined : this.props.onClick,
            tabIndex: this.props.onClick && !this.props.disabled ? 0 : undefined,
            role: this.props.onClick && !this.props.disabled ? 'button' : undefined,
            ref: this.props.innerRef
        };
        const glyphClassName = classNames(styles.glyph, this.props.glyphClassName, {
            [styles.clickable]: this.props.onClick,
            [styles.controlled]: this.props.controlled,
            [styles.active]: this.props.active,
            [styles.disabled]: this.props.disabled,
            [styles[this.props.size]]: this.props.size
        });

        // the key prop in svg need for fixing the IE issue related to external content in SVG
        // https://stackoverflow.com/questions/22516712/svg-use-tag-with-external-reference-in-ie-11
        return (
            <i {...iconProps}>
                <svg key={this.props.glyph} className={glyphClassName}>
                    <use xlinkHref={`#${this.props.glyph}`} />
                </svg>
            </i>
        );
    }
}

Icon.propTypes = {
    glyph: string.isRequired,
    size: oneOf(values(SIZES)),

    controlled: bool,
    active: bool,
    disabled: bool,

    className: string,
    glyphClassName: string,

    innerRef: func,
    onClick: func
};

Icon.defaultProps = {
    size: SIZES.MEDIUM,

    controlled: true,
    active: false,
    disabled: false,

    className: undefined,
    glyphClassName: undefined,

    innerRef: undefined,
    onClick: undefined
};

export default Icon;
```

```SCSS
// icon.scss
@import '../scss/theme';

.icon {
  display: inline-flex;
}

.glyph {
  fill: currentColor;

  &:hover:not(.controlled) {
    fill: $c-french-blue;
  }
}

.clickable {
  cursor: pointer;
}

.active {
  fill: $c-french-blue;
}

.disabled {
  color: $c-pastel-gray;
}

.mini {
  width: $gs-mini;
  height: $gs-mini;
}

.tiny {
  width: $gs-tiny;
  height: $gs-tiny;
}

.small {
  width: $gs-small;
  height: $gs-small;
}

.medium {
  width: $gs-medium;
  height: $gs-medium;
}

.large {
  width: $gs-large;
  height: $gs-large;
}

.big {
  width: $gs-big;
  height: $gs-big;
}

.huge {
  width: $gs-huge;
  height: $gs-huge;
}

.massive {
  width: $gs-massive;
  height: $gs-massive;
}

.banner {
  width: $gs-banner;
  height: $gs-banner;
}

// _sizes.scss
// Glyph sizes
$gs-mini: 10px;
$gs-tiny: 12px;
$gs-small: 14px;
$gs-medium: 16px;
$gs-large: 18px;
$gs-big: 20px;
$gs-huge: 22px;
$gs-massive: 24px;
$gs-banner: 48px;

// Border radius
$b-radius: 3px;
```